apps/builder/app/builder/features/breakpoints/use-update-canvas-width.ts

```diff
@@ -27,7 +27,10 @@ export const useUpdateCanvasWidth = () => {
     if (isPreviewMode || selectedBreakpoint === undefined) {
       return;
     }
-    const maxWidthBelowNextBreakpoint = (nextBreakpoint?.minWidth ?? 0) - 1;
+    const maxWidthBelowNextBreakpoint =
+      (nextBreakpoint && "minWidth" in nextBreakpoint
+        ? nextBreakpoint.minWidth ?? 1
+        : 1) - 1;
```

Comment:
Split the logic inside parentheses into separate variable and add some comments. The logic I getting less clear now.